tests/framework/framework.go

```diff
@@ -213,6 +214,8 @@ func DeleteNS(c *kubernetes.Clientset, ns string) error {
 
 // Gets an instance of a kubernetes client that includes all the CDI extensions.
 func (f *Framework) GetCdiClient() (*cdiClientset.Clientset, error) {
+	fmt.Println(f.Master)
```

Comment:
is this debugging? If not then you want to use the GinkgoWriter is seen in other framework uses of `fmt.Fprintf`

Reply:
It was debugging, I forgot to remove them.